### ISSUE TITLE
ServersList not working

### ISSUE THREAD

**Issue by otDan** [DaBluLite/css-snippets#11]
After the latest update it seems that the theme is straight up ignored

**Comment by otDan** [comment]
seems to at least be working now but the images are not scaled: 
![image](https://github.com/DaBluLite/css-snippets/assets/59731738/7387d214-5530-433f-a187-97d971b43e55)

**Comment by otDan** [comment]
is there anything planned to fix or just leaving it as is?

**Comment by DaBluLite** [comment]
I can't seem to reproduce this result
Are you on discord stable?

**Comment by otDan** [comment]
> I can't seem to reproduce this result Are you on discord stable?

yes, has been like this since the update before I created the issue

**Comment by DaBluLite** [comment]
I don't know what to tell you, the snippet is working fine for me since I updated it 2 weeks ago

**Comment by otDan** [comment]
> I don't know what to tell you, the snippet is working fine for me since I updated it 2 weeks ago

I've tried in web, discord client, new client, other browser, it's always like this:

![image](https://github.com/user-attachments/assets/0a6a4152-8dde-4de4-afe1-c071f57eb651)

**Comment by otDan** [comment]
![image](https://github.com/user-attachments/assets/34b94771-90a0-4344-8d1e-ec7c1c8f90dd)

the image is always cut out at half and the server button is pressable only on the image

**Comment by otDan** [comment]
I have no other themes, so yeah I don't know what to tell you I've reporduced this every time I tried using the snippet

**Comment by otDan** [comment]
it seems that the height propriety does not actually resize anything besides the container
`--guild-list-height: 24px;`

**Comment by DaBluLite** [comment]
I see, I will take a look at it later. Tbh, this snippet is already held by hopes and dreams, so I might just cut it out completely

**Comment by otDan** [comment]
welp it stopped working again, not sure if you wanna still support it but it's the only thing that makes discord usable with the amount of servers I am in lol
